src/js/ko-components/view-models/orifice-calculator.coffee

```diff
@@ -19,6 +17,12 @@ class OPL.KoComponents.ViewModels.OrificeCalculator
         when 200 < @operatingPressure() <= 400 then OPL.OrificeCalculator.Config.Dictionaries.Messages.operatingPressureWarningMayResult
         when 401 <= @operatingPressure() then OPL.OrificeCalculator.Config.Dictionaries.Messages.operatingPressureWarningWillResult
 
+    @pipeID = ko.observableArray _.map OPL.OrificeCalculator.Config.Dictionaries.PipeID, (pipeName, pipeDiameter) ->
+      name: pipeName
+      value: pipeDiameter
+
+    @selectedPipeID = ko.observable 1.939
```

Comment:
Why would we even have a dictionary if then we refer to the values magically in the code?